content/en/threat-model.md

```diff
@@ -0,0 +1,123 @@
+|                        |                  | x             |                       | x            |               |              |             | Same as Fulcio CT log + Rekor server: can now use the replayed certs created by Fulcio CT log because Rekor will not report that they are already in the log, even with more sophisticated monitors.                                                                                                                                                                                                                                                                                                                                                     |
+|                        |                  |               |                       | x            | x             |              |             | Same as Rekor server, but changes to the CT log would not be reported by the monitor, would make recovery of the log much harder because the attacker could alter the historical entries, may require re-signing all metadata as previous timestamps are untrusted.                                                                                                                                                                                                                                                                                      |
+|                        |                  | x             | x                     | x            |               |              |             | Same as [Fulcio CT log + Rekor server], but replay and fork attacks will not be reported by monitors.                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|                        | x                |               |                       |              |               |              |             | Arbitrary software attack: Can issue certs for any OIDC issuer/identity and use those to sign any software desired. Bob will see these certificates as valid as they are signed by the Fulcio CA and included in the Fulcio CT log. To recover, the TUF root of trust for Fulcio needs to be invoked to revoke trust in the compromised Fulcio CA server.  If the Fulcio CT log is working, then you cannot change the history  or block issuance / modification of past things (which is not that relevant given the ability to issue keys, etc. now),  |
```

Comment:
Is it possible to preserve existing signatures prior to the compromise based on the CT logs (if you have a date of compromise)?

Reply:
This is what the last sentence is trying to say, I updated the wording to try to make this more clear